experimental/packages/otlp-exporter-base/src/platform/browser/OTLPExporterBrowserBase.ts

```diff
@@ -52,14 +52,6 @@ export abstract class OTLPExporterBrowserBase<
     }
   }
 
-  onInit(): void {
-    _globalThis.addEventListener('unload', this.shutdown);
-  }
```

Comment:
this is still required by the abstract base (causes compile to fail). you can change this to an empty implementation and add a comment explaining why it is empty. :slightly_smiling_face: